### DISCUSSION TITLE
python执行 python setup.py install 报错 error: can't copy '../configs': doesn't exist or not a regular file

### DISCUSSION THREAD

**Discussion by AyanamiRay** [mlc-ai/mlc-zh#22] [Q&A]
麻烦问下，python执行   python setup.py install 报错 error: can't copy '../configs': doesn't exist or not a regular file 如何处理？

**Comment by Hzfengsy** [comment]
请使用 `pip install mlc-ai-nightly -f https://mlc.ai/wheels`进行安装

**Reply by AyanamiRay** [reply]
谢谢
